services/github/gist/github-gist-last-commit.service.js

```diff
@@ -11,17 +12,18 @@ const schema = Joi.object({
 export default class GistLastCommit extends GithubAuthV3Service {
   static category = 'activity'
   static route = { base: 'github/gist/last-commit', pattern: ':gistId' }
-  static examples = [
-    {
-      title: 'GitHub Gist last commit',
-      namedParams: {
-        gistId: '8710649',
+  static openApi = {
+    '/github/gist/last-commit/{gistId}': {
+      get: {
+        summary: 'GitHub Gist last commit',
+        description: documentation,
+        parameters: pathParams({
+          name: 'gistId',
+          example: '8710649',
+        }),
       },
-      staticPreview: this.render({ commitDate: '2022-07-29T20:01:41Z' }),
-      keywords: ['latest'],
```

Comment:
I may have missed something, we're dropping keywords in this example and a few others. Are these not supported anymore?

Reply:
Good question.

With our old search, it could only search some limited fields.

The OpenAPI format doesn't have "keywords", but the search plugin we use now does give us full text search over all text on the page (so that includes the summary, description and all parameter fields, including param names, descriptions, etc). So yes we are losing `keywords` but we have gained more powerful search. For example https://shields.io/search?q=vcsType shows me all badges that take a param called `vcsType`, even though none of them have it in their title or description. We didn't have this before.

I think in the grand scheme of things, the best way to deal with this is to try and make sure that if we're removing keywords that would help users search for things and they're not covered by other text, we make sure we're adding a description that includes them. This is something I've overlooked in some cases. For example, https://shields.io/search?q=ruby doesn't return results for any of our ruby gems badges because they're all just called "gem [something]". I think in those cases, we can usefully add a description that also includes the word "ruby" to help with that.

I'm happy to take an action to review any we've already merged or got open for this.